pages/glossary/semantic-role.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Semantic Role
 key: semantic-role
+unambiguous: true
+objective: true
 ---
 
 A semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role) is a semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
```

Comment:
Wondering if, to be unambiguous, this definition within ACT Rules needs to be clear that a role should be one of those defined by the WAI-ARIA spec role taxonomy (https://www.w3.org/TR/wai-aria-1.1/#role_definitions). Even for custom elements.

Reply:
I don't think this is needed to make the def unambiguous.

Additionally, roles can be defined somewhere else (e.g., the Graphics AAM defines new roles: https://w3c.github.io/graphics-aam/#mapping_role_table).

Lastly, linking to the WAI-ARIA roles kinda ties down to the HTML technology. A semantic role could be defined for pdf, for LaTeX, or any other kind of document and these technologies could choose a different taxonomy of roles without this being intrinsically bad.

We could use add a note to point to the list, it is indeed a valuable addition to the definition.
Can you do that? 😁

Reply:
I'll make it separate to this PR.

Reply:
https://github.com/act-rules/act-rules.github.io/issues/1129